cmd/s3-proxy/main.go

```diff
@@ -133,3 +136,29 @@ func main() {
 		logger.Fatal(err)
 	}
 }
+
+func main() {
+	var configFolder string
+
+	var rootCmd = &cobra.Command{
+		Use:   "s3-proxy",
+		Short: "S3 Reverse Proxy",
+		Long:  "S3 Reverse Proxy with GET, PUT and DELETE methods and authentication (OpenID Connect and Basic Auth)",
+		Run: func(cmd *cobra.Command, args []string) {
+			startServer(configFolder)
+		},
+	}
+
+	var versionCmd = &cobra.Command{
+		Use:   "version",
+		Short: "Print the version number of s3-proxy",
+		Run: func(cmd *cobra.Command, args []string) {
+			v := version.GetVersion()
+			fmt.Printf("version: %s (git commit: %s) built on %s", v.Version, v.GitCommit, v.BuildDate)
+		},
+	}
+
+	rootCmd.AddCommand(versionCmd)
+	rootCmd.PersistentFlags().StringVar(&configFolder, "config", "conf/", "Config folder (default is $PWD/conf/)")
```

Comment:
As the tool is also built for Windows and Mac, can we change this:
```(default is $PWD/conf/)```
to something like this:
```(default is <Current Working Directory>/conf/)```
?
What do you think ?

Reply:
Yes ! I'll correct this right now.